tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/DebugAdapterBaseTestCase.java

```diff
@@ -471,15 +471,15 @@ protected void assertEvaluationError(StoppedEventArguments context, String expre
     /**
      * Can be used to assert stack frame name, line and source.
      *
-     * @param frames   debug hit stack frames.
+     * @param frame   debug hit stack frame.
```

Comment:
shall we rectify the formatting issue in here?

Reply:
Formatted